WolvenKit.App/ViewModels/Tools/ImportExportItemViewModel.cs

```diff
@@ -45,11 +43,7 @@ private void Properties_PropertyChanged(object? sender, System.ComponentModel.Pr
 
     public void SetProperties(ImportExportArgs args)
     {
-        Properties.PropertyChanged -= Properties_PropertyChanged;
```

Comment:
Not sure why you removed that.
Added that last week (e66d87e) to reflect the changes in the file info column and preserve the binding.

Reply:
Are those changes merged back? Because it didn't work for me, and putting the listener back makes it work O_O

Reply:
Yeah, its in main. And it worked for me.

Reply:
Maybe I broke it in another spot because the way it is is working for me right now O_O Let's talk on Discord